Justfile

```diff
@@ -257,11 +262,11 @@ generate-provider-sdk schema-name schema-version:
 
 # Compiles the previously generated scala API code for the given provider, e.g. `just compile-provider-sdk kubernetes 4.0.0`
 compile-provider-sdk schema-name schema-version:
-	scala-cli --power compile {{codegen-output-dir}}/{{schema-name}}/{{schema-version}} --suppress-experimental-feature-warning
+	scala-cli --power compile {{scala-cli-bloop-opts}} {{codegen-output-dir}}/{{schema-name}}/{{schema-version}} --suppress-experimental-feature-warning --interactive=false
```

Comment:
these opts will be ignored if bloop is already running

Reply:
should we `scala-cli bloop exit` before compiling then, WDYT?